src/Pages/Main/Main.js

```diff
@@ -0,0 +1,402 @@
+                          styles.transactionIndicator,
+                          {backgroundColor: '#967AF3'},
+                        ]}
+                      />
+                      <Text style={styles.transactionTitle}>$2,730</Text>
+                    </View>
+                  </View>
+                </View>
+              </CardBody>
+
+              <CardFooter>
+                <View>
+                  <Text style={[styles.text, {marginBottom: 10}]}>
+                    You spent $2,732 on dining out this month. Let’s try to make
+                    it lower
+                  </Text>
+                  <Text
+                    style={[
+                      styles.text,
+                      {color: '#7165E3', textDecorationLine: 'underline'},
+                    ]}>
+                    Tell me more
+                  </Text>
+                </View>
+              </CardFooter>
+            </Card>
+          </View>
+
+          <View>
+            <Text style={styles.title}>Activity</Text>
+            <View style={styles.activityFlex}>
+              <Card style={styles.rowCard}>
+                <View style={styles.activityIconHolder}>
+                  <Icon style={styles.activityIcon} name="send" />
+                </View>
+                <Text style={styles.text}>Transfer</Text>
+              </Card>
+
+              <Card style={styles.rowCard}>
+                <View style={styles.activityIconHolder}>
+                  <Icon style={styles.activityIcon} name="credit-card-alt" />
+                </View>
+                <Text style={styles.text}>My Card</Text>
+              </Card>
+
+              <Card style={styles.rowCard}>
+                <View style={styles.activityIconHolder}>
+                  <Icon style={styles.activityIcon} name="bar-chart" />
+                </View>
+                <Text style={styles.text}>Insight</Text>
+              </Card>
+            </View>
+          </View>
+
+          <View>
+            <Text style={styles.title}>Complete Verification</Text>
+
+            <Card>
+              <View style={styles.verifyFlex}>
+                <Text style={styles.verifyFlexTitle}>{verifyPercent}%</Text>
+                <Text style={styles.verifyFlexText}>7 of 10 completed</Text>
+              </View>
+
+              <View style={styles.verifyBar}>
+                <View
+                  style={[styles.verifyProgress, {width: `${verifyPercent}%`}]}
+                />
+              </View>
+              <Divider style={{marginBottom: 15}} />
+              <View style={styles.row}>
+                <Icon name="user" style={styles.verifyIcon} />
+                <View>
+                  <Text style={styles.subTitle}>Personal Information</Text>
+                  <Text style={[styles.text, {marginBottom: 17}]}>
+                    When you register for an account, we collectt personal
+                    informmation
+                  </Text>
```

Comment:
This Text component overflowed the Card component.

please it is "we collect" and not "we collectt"